maths/greater_common_divisor.py

```diff
@@ -1,25 +1,25 @@
-"""
-Greater Common Divisor.
+# Python program to find the G.C.D of two input number
 
-Wikipedia reference: https://en.wikipedia.org/wiki/Greatest_common_divisor
-"""
+# define a function
+def computeGCD(x, y):
```

Comment:
What about __greatest_common_denominator()__??

Reply:
It's Greatest Common Divisor Code